src/components/UI/ButtonStyled/BookmarkButton.styled.js

```diff
@@ -10,9 +10,8 @@ export const BookmarkButton = styled.button`
 export const WrapperBookmark = styled.section`
 	display: flex;
 	justify-content: space-evenly;
-	width: 80vw;
+	width: 100vw;
```

Comment:
100% is saver here, in firefox this should break the styling of your app

Reply:
Fixed